backend/trivia_api/views.py

```diff
@@ -61,6 +61,71 @@ def users(user_id):
     return dummy_users
 
 
+@blueprint.route("/signup", methods=["GET", "POST"])
```

Comment:
```suggestion
@blueprint.route("/signup", methods=["POST"])
```

no need to allow get in the first place